src/test/java/org/jenkinsci/plugins/workflow/multibranch/JobPropertyStepTest.java

```diff
@@ -211,4 +230,135 @@
         r.assertBuildStatusSuccess(r.waitForCompletion(b4));
     }
 
+    @Issue("JENKINS-34005")
+    @Test public void triggersProperty() throws Exception {
+        WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "p");
+        // Verify the base case behavior.
+        p.setDefinition(new CpsFlowDefinition("echo 'foo'"));
+
+        assertTrue(p.getTriggers().isEmpty());
+
+        r.assertBuildStatusSuccess(p.scheduleBuild2(0));
+
+        // Make sure the triggers are still empty.
+        assertTrue(p.getTriggers().isEmpty());
+
+        // Now add a trigger.
+        p.setDefinition(new CpsFlowDefinition(
+                (HAVE_SYMBOL ?
+                        "properties([pipelineTriggers([\n"
+                                + "  cron('@daily'), [$class: 'MockTrigger']])])\n" :
+                        "properties([pipelineTriggers(\n"
+                                + "  triggers: [[$class: 'TimerTrigger', spec: '@daily'],\n"
```

Comment:
Ah, since we have `@Symbol` on `pipelineTriggers`, its parameters are covered too. Fixing.